buf/registry/module/v1beta1/label_service.proto

```diff
@@ -170,7 +170,9 @@ message CreateOrUpdateLabelsRequest {
     LabelRef label_ref = 1 [(buf.validate.field).required = true];
     // The id of the Commit to associate with the Label.
     //
-    // If the Label already existed, the Label will now point to this Commit.
+    // If the Label already existed, the Label will now point to this Commit, as long as this Commit
```

Comment:
This documentation should be copied and/or moved to the RPC documentation  - we should explain that you cannot update labels to earlier commits

Reply:
Copied the docs to the RPC.